### PR TITLE
docs: use correct example for `useBell` with a specific store

### LINKED ISSUE #72: Is the documentation for `useBell` correct?
When I was trying to use `useBell` hook with a specified id for the store, it threw and error. Looking at the [source code](https://github.com/magicbell-io/magicbell-js/blob/4fdf93790ef81a17ef7b8462917de3d9fcd387ae/packages/react-headless/src/hooks/useBell.ts#L3), shouldn't it be something like this?

```tsx
function NotificationsList() {
  const { unreadCount, markAllAsSeen } = useBell({ storeId: 'unread' });   // <- fix

  return (
    <button onClick={() => markAllAsSeen()}>
      <span>{unreadCount}</span>
    </button>
  );
}
```

example
| doesn't work | works |
| --- | --- |
| image |  image |

### PR DESCRIPTION
**Pull request by SuzukiRyuichiro**
fix #72

### CODE REVIEW

**Comment by unamashana** [comment]
Thank you @SuzukiRyuichiro! @smeijer will review this shortly.

File: .changeset/clean-tigers-destroy.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@magicbell/react-headless': patch
+---
+
+The documentation example for useBell was throwing a type error when implementing. Looking at the source code, it should pass an object with key storeId, instead of straight up giving the string
```
Comment (smeijer) [inline .changeset/clean-tigers-destroy.md]:
```suggestion
Fix type annotation for `useBell` arguments to accept an object instead of string.
```

**Review by smeijer — APPROVED** [review]

**Comment by smeijer** [comment]
Thanks @SuzukiRyuichiro! Much appreciated.

File: .changeset/clean-tigers-destroy.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@magicbell/react-headless': patch
+---
+
+Fix type annotation for `useBell` arguments to accept an object instead of string.
```
Comment (smeijer) [inline .changeset/clean-tigers-destroy.md]:
```suggestion
Fix example of in docs for `useBell`, to provide `storeId` via an options object.
```

**Pull request merged**